src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -126,36 +142,69 @@ static void compoundValueBson(std::vector<orion::CompoundValueNode*> children, B
   }
 }
 
+/* ****************************************************************************
+*
+* bsonAppendAttrValue -
+*
+*/
+void bsonAppendAttrValue(BSONObjBuilder& bsonAttr, ContextAttribute* caP)
+{
+  switch(caP->valueType)
+  {
+    case ValueTypeString:
+      bsonAttr.append(ENT_ATTRS_VALUE, caP->stringValue);
+      break;
+    case ValueTypeNumber:
+      bsonAttr.append(ENT_ATTRS_VALUE, caP->numberValue);
+      break;
+    case ValueTypeBoolean:
+      bsonAttr.append(ENT_ATTRS_VALUE, caP->boolValue);
+      break;
+    default:
+      LM_E(("Runtime Error (unknown attribute type)"));
+  }
+}
```

Comment:
An empty line between break and the next case (or default:) makes the code easier to read (and it follows the 'de facto standard' of the rest of the proyect).